modules/test/src/main/java/org/apache/fluo/recipes/test/AccumuloExportITBase.java

```diff
@@ -39,7 +39,8 @@
 /**
  * This class is intended to be extended by classes testing exporting from Fluo to Accumulo. Using
  * MiniFluo by itself is easy. However, using MiniAccumulo and MiniFluo together involves writing a
- * lot of boiler plate code. Thats why this class exists, its a place to put that boiler plate code.
+ * lot of boiler plate code. That's why this class exists, its a place to put that boiler plate
```

Comment:
boilerplate is one word. There's an opportunity to clean that up here also, which would probably keep "code" on the same line.

Also, "exists, its a place" should be replaced with "exists: it's a place"